multihost_test/Bugzillas/test_regression.py

```diff
@@ -292,3 +293,53 @@ def test_bz_973647(self, multihost):
             client.run_command("grep hildegarda /etc/passwd && userdel hildegarda")
             client.run_command("rm -rf /althome")
             client.run_command("semanage fcontext -d '/althome(/.*)*'")
+
+    def test_bz_921995(self, multihost):
+        """
+        :title: Include upstream patches to make it clear in the
+            error message why userdel wasn't able to delete an account
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=921995
+        :id: 784fae50-170a-11ee-9160-845cf3eff344
+        :steps:
+          1. Create a new user with the username specified
+          2. Sets the password for the newly created user
+          3. Check user can login
+          4. Try to delete the user created
+          5. Deletes the user forcefully, including their home
+            directory and any files owned by them.
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should succeed
+          4. Should not succeed
+          5. Should succeed
+        """
+        user = "test921995"
+        client = multihost.client[0]
+        client.run_command(f"useradd {user}")
+        client.run_command(f"echo {user} | passwd --stdin {user}")
+        ssh = SSHClient(multihost.client[0].ip, user, 'test921995')
+        ssh.connect()
+        with pytest.raises(subprocess.CalledProcessError):
+            client.run_command(f"LANG=c userdel {user}")
+        ssh.close()
+        client.run_command(f"userdel -rfZ {user}")
+
+    def test_bz_782515(self, multihost):
```

Comment:
I'm kind of surprised by this test case, as it doesn't exercise the problem the bugzilla mentions, but it must be that way for some reason. Taking the test case as the source of truth, do you mind changing the title so that they match? My suggestion: `useradd is able to create homedir if top-level directory exists`

Reply:
Done

Reply:
But now after fixing user is able to create homedir if top-level directory exists, before it does not . So i think tittle is correct . Though i have changed it to as you suggest

Reply:
By this line :
 with pytest.raises(subprocess.CalledProcessError):
            client.run_command(f"ls -l {temp_dir}")

It just checking that the directory does not exists from before